cmake/Buildergocub-software.cmake

```
@@ -3,7 +3,6 @@
 
 include(YCMEPHelper)
 
```

```suggestion

find_or_build_package(YARP QUIET)
```

See https://github.com/robotology/robotology-superbuild/blob/master/doc/developers-faqs.md#how-to-add-a-new-package .